matlab-src/whole_body_model_functions/wbm_getWorldFrameFromFixedLink.m

```diff
@@ -0,0 +1,33 @@
+function [ world_R_base,world_p_base ] = wbm_getWorldFrameFromFixedLink( varargin )
+%WBM_GETWORLDFRAMEFROMFIXEDLINK returns the floating base position for a
+%given reference link name and joint configuration
+%
+%   Arguments :
+%       Optimised Mode : link_name - string matching URDF name of the link (frame)
+%       Normal Mode : link_name - string matching URDF name of the link (frame)
+%                      qj - joint configuration ( (NumDoF x 1))
+%   Returns : R - rotation from rootLink to world frame (3 x 3)
+%             p - translation from rootLink to world frame (3 x 1) 
+%
+% Author : Naveen Kuppuswamy (naveen.kuppuswamy@iit.it)
+% Genova, Dec 2014
+
+    switch(nargin)
+        case 1 
+            qT = wbm_forwardKinematics(varargin{1});
+            p = zeros(3,1);
+            [base_p_reference,base_R_reference] = frame2posrot(qT);
+            world_p_base = -base_p_reference;
```

Comment:
base_p_reference is supposed to be expressed in base orientation, while world_p_base should be expressed in world orientation, instead here we are just changing the sign without any rotation.